enterprise-suite/templates/alertmanager.yaml

```diff
@@ -1,3 +1,18 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: alertmanager-storage
+spec:
+  storageClassName: {{ default .Values.defaultStorageClass .Values.esAlertmanagerStorageClass }}
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ .Values.alertmanagerVolumeSize }}
+  parameters:
+    fstype: xfs
```

Comment:
Is the "prometheus uses lots of inodes" problem still true? The TSD has been radically re-written in 2.x, I thought the "block per timeseries" thing was gone and backing store uses a small number of large files, indexing, etc.

Reply:
Based on my experience with running out of inodes on ext4, I think xfs is the safest option regardless. It's also the default filesystem in RHEL if you're concerned about its stability.

Reply:
Also, ext4 has reliability issues on RHEL in my experience.

Reply:
xfs is cool, I just want to leave this selection up to the user. I don't think we need to force it and can let users try other things if they want. Prometheus 2.x shouldn't have the same urgent problem with inodes, as it only has a few files. Prometheus 1.x had a huge problem here, with a file per timeseries, but we aren't using that.

My minikube install, just poking around, has 19 files in prometheus /data, for 4,895 timeseries. The kernel has 100k inodes in use for all the kubernetes things, but prometheus is using only 28 of those. It seems a tiny amount.

```
root@prometheus-server-fffb9b9cf-rj2tn:/# ls /proc/1/root/proc/1/fd | wc -l
28
```

Reply:
I doubt a user will care until their volume dies.